.NET/Microsoft.Recognizers.Definitions/French/DateTimeDefinitions.cs

```diff
@@ -196,7 +196,7 @@ public static class DateTimeDefinitions
 		public const string InexactNumberRegex = @"\b(quelque|quel qu[ée]s|quelqu[ée]s|plusieur|plusieurs|divers)\b";
 		public static readonly string InexactNumberUnitRegex = $@"({InexactNumberRegex})\s+({DurationUnitRegex})";
 		public static readonly string RelativeTimeUnitRegex = $@"(((({ThisPrefixRegex})?)\s+({TimeUnitRegex}(\s*{NextSuffixRegex}|{PastSuffixRegex})?))|((le))\s+({RestrictedTimeUnitRegex}))";
-		public static readonly string RelativeDurationUnitRegex = $@"(((?<=({ThisPrefixRegex})\s+)?\b({DurationUnitRegex})(\s+{NextSuffixRegex}|{PastSuffixRegex})?)|((le|my))\s+({RestrictedTimeUnitRegex}))";
+		public static readonly string RelativeDurationUnitRegex = $@"((\b({DurationUnitRegex})(\s+{NextSuffixRegex}|{PastSuffixRegex})?)|((le|my))\s+({RestrictedTimeUnitRegex}))";
```

Comment:
I understand the reason for these changes.
But I also notice that this Regex is not the same implementation as the English version. I'm not quite familiar with French and Italian, is this because of some language specific reasons?
This change would make this Regex can't support cases like "current hour" in French and Italian.
I agree that this is not quite common cases, but can we find a solution to also include "ThisPrefixRegex" in this method? Maybe reference the changes in #972?

Thanks.

Reply:
Hi @guom08. First of all, sorry for the delay in answering. I'm not quite familiar with French and Italian, but I assume that the differences between the English and French/Italian version of this Regex are related to the language grammar rules. Regarding the changes in the French and Italian implementation of this Regex, there will be no problems on supporting cases like "current hour/minute/day/week/month/etc", mainly because checking the tests related to this regex (DateTimeExtractor, DateTimeModel, and DateTimeParser) there are no test which cover this definition, but also because there are more than one regex that covers these cases. For example, here are some screenshots of the successful execution of a possible scenario of this regex (`Je serai dehors dans cette semaine`---> `I'll be out this week`; defined in [DatePeriodExtractor](https://github.com/Microsoft/Recognizers-Text/blob/master/Specs/DateTime/French/DatePeriodExtractor.json#L135)), both in JS and C# versions:

![image](https://user-images.githubusercontent.com/43762887/50523976-eb9dfe80-0ab1-11e9-963f-171f472da13b.png)

![image](https://user-images.githubusercontent.com/43762887/50523977-ee98ef00-0ab1-11e9-904f-989a5644f02e.png)

Reply:
I see, thanks.
But for this PR, can we wait for the discussion of PR #1069? As NodeJS version 10.30 can support look behind native.

If this PR is urgent to merge, please let me know.